front_end/views/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
```

Comment:
This is not a front end file - this is something that should be returned by the server?